packages/experience-builder-sdk/src/blocks/editor/VisualEditorBlock.tsx

```diff
@@ -121,9 +121,26 @@ export const VisualEditorBlock = ({
           const [, uuid, ...path] = variableMapping.path.split('/');
           const binding = dataSource[uuid] as Link<'Entry' | 'Asset'>;
 
-          const boundValue = areEntitiesFetched
+          let boundValue: string | Link<'Asset'> | undefined = areEntitiesFetched
             ? entityStore.current?.getValue(binding, path.slice(0, -1))
             : undefined;
+
+          // If there is no boundValue found, we check with the second last item removed as well
+          // this might be the case for the additional linked asset
```

Comment:
"we check with the second last item removed as well"
I don't really get what this means, sorry ^^